select.js

```diff
@@ -0,0 +1,14 @@
+define(function() {
+    var init = function() {
+        $('body').on('change', '.c-select', function() {
```

Comment:
Good question --- the adaptivejs infrastructure surrounding JS initialization for components looks at a "stencil" data-attribute, which authors could use themselves. We do this within adaptivejs so we can easily search for all `[data-stencil]` elements on the page. 

That said, I see an argument for authors using whatever class they've already defined for styling, i.e. `c-select` in this circumstance. I don't see an issue with it as written.